type=class
superclass=Net=ProtocolError
included=
extended=
dynamically_included=
dynamically_extended=
library=net.protocol
aliases=Net=ProtocRetryError
aliasof=

なんらかの再試行をすることで、処理の継続が可能であるような
エラーが発生した場合に発生する例外です。
